.github/workflows/deploy-sphinx-docs-to-ghpages.yml

```
@@ -8,10 +8,14 @@ on:
         python-version:
           required: true
           type: string
-        path-artifact:
+        path-doc:
           required: false
           type: string 
-          default: ./doc/build/html
+          default: ./doc
+        Cached-package:
```

Use only `kebab-case` convention (in general, never mix case conventions in the same group of variables).

Moreover, this is not completely descriptive about the variable, but about how the variable is used. For the user, it is more intuitive to describe what is expected, since they don't have to read the workflow to use it (like we are not reading all the actions code we are using).
```suggestion
        package-manager:
```